### ISSUE TITLE
OCRmyPDF silently fails on input filenames like uppercase *.PDF

### ISSUE THREAD

**Issue by Wikinaut** [jbarlow83/OCRmyPDF#20]
My new duplex scanner is BROTHER ADS-2600we. It generates PDF (which are not compatible and make also ```convert``` fail. It can however generate PDF/A. The standard filenames have the form

[0-9]{8}.PDF

Example: 06091500.PDF, 06091501.PDF etc. for files scanned on 06. September 2015. These filenames (I don't like the format) cannot be changed in the scanner.

##### Problem:

When you start
```
ocrmypdf --verbose -L log.txt -l deu 06091500.PDF 06091500.ocr.pdf
```
this silently fails ! ("...No jobs were run because no file names matched.")

Workaround:
Rename files 06091500.PDF to x.pdf and process then.

**Comment by jbarlow83** [comment]
It was doing a case sensitive search for files name `.pdf` not `.PDF`. Fixed in develop branch, will push a fix.

**Comment by jbarlow83** [comment]
Fixed v3.1
